src/app/projects/add-edit-project/add-edit-project.component.html

```diff
@@ -298,8 +314,15 @@ <h2>Engagement Information</h2>
         </div>
         <div class="flex-container form-container-65 flex-column">
           <label for="enagementInfo">Engagement Tab Information </label>
-          <ckeditor name="engagementInfo" id="engagementInfo" formControlName="engagementInfo" [editor]="Editor"
-            [data]="project?.engagementInfo" (ready)="editorOnReady($event)"></ckeditor>
+          <ckeditor
+            name="engagementInfo"
+            id="engagementInfo" 
+            formControlName="engagementInfo"
+            [editor]="Editor"
+            [data]="project?.engagementInfo"
+            [config]=" isEditing ? {} : noImageToolbarConfig"
```

Comment:
I see, so we remove the ability to insert images until we have a project ID?

Reply:
Exactly, trying to add images before a project is valid can get us in to trouble. Like other file upload spots (shapefile, logos), I've removed this one until the project is saved for the first time.